boudicca.base/publisher-event-html/src/main/resources/templates/index.hbs

```diff
@@ -17,5 +17,6 @@
 
         <script src="js/main.js"></script>
         <script src="js/menu.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
```

Comment:
please don't use CDNs but just add it into our static folder, it is not that big and we avoid external depenencies on our website, which can add instability + data leaks